src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -104,6 +104,11 @@ case class AppDefinition(
     container.fold(Seq.empty[Volume])(_.volumes).collect{ case vol: PersistentVolume => vol }
   }
 
+  /**
+    * @return the disk resources required for volumes
+    */
+  def volumeDiskResources: Double = persistentVolumes.map(_.persistent.size).sum.toDouble
```

Comment:
since the existing resource property is called `disk`, I'm inclined to call this `diskForVolumes`.